docs/releases.rst

```diff
@@ -29,18 +18,38 @@ https://tmt.testing-farm.io/ and can be used together with the
 ``tmt link`` command to :ref:`link-issues` with related tests and
 plans.
 
+Files submitted during a test via
+:ref:`tmt-file-submit</stories/features/report-log>` or BeakerLib's
+``rlFileSubmit`` are now a part of main test result as logs.
+
 The ``--environment`` option now properly supports dotenv files as
 documented. Previously, while the help text indicated that both YAML
 and dotenv formats were supported, only YAML files worked correctly.
 Now dotenv files can be used with the ``@filename.env`` syntax.
 
+The :ref:`/plugins/report/reportportal` report plugin now supports
+a new ``upload-log-pattern`` key. This key allows users to select
+which logs should be uploaded by specifying the pattern to search
+for in the log file names. Check result logs are also affected by
+this key but are uploaded only if the check fails or if an error
+occurs during execution.
+
+Security improvements have been made to prevent tmt scripts
```

Comment:
Not completely sure these are actually security improvements...

Reply:
thanks, removed the `Security` from the sentence